src/calculateRentalCost.js

```diff
@@ -3,8 +3,21 @@
  *
  * @return {number}
  */
+
+const seven = 7;
+const three = 3;
+const saleForThreeDays = 20;
+const saleForSevenDays = 50;
+const startPrice = 40;
+
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= seven) {
+    return days * startPrice - saleForSevenDays;
+  } else if (days >= three) {
+    return days * startPrice - saleForThreeDays;
+  }
```

Comment:
There is no need to use the `else` keyword after the `return` statement. When the `return` statement is executed, the function stops its execution and returns the specified value. So, if the condition in the `if` statement is met, the function will return the value and will not check the other conditions. Here is a better way to structure this code:

js
if (days >= LONG_TERM_RENTAL) {
  return days * startPrice - LONG_TERM_DISCOUNT;
}

if (days >= SHORT_TERM_RENTAL) {
  return days * startPrice - SHORT_TERM_DISCOUNT;
}

return days * startPrice;
```